conda/recipes/cudf_kafka/meta.yaml

```diff
@@ -41,13 +44,21 @@ requirements:
     - {{ compiler('cxx') }}
     - ninja
     - sysroot_{{ target_platform }} {{ sysroot_version }}
+    # TODO: Remove after https://github.com/rapidsai/cudf/pull/14292 updates
+    # cudf_kafka to use scikit-build
+    {% if cuda_major == "12" %}
+    - cuda-gdb
```

Comment:
I think we can drop `cuda-gdb` from this dependency list if we're defining `CUDA_HOME`, since the only reason cudf_kafka uses `cuda-gdb` is to determine `CUDA_HOME` if that environment variable is not set.

Reply:
Yes, that's correct. I had originally added cuda-gdb before realizing that the cuda-gdb-based detection in setup.py was not compatible with the conda-forge CUDA package layout anyway and therefore installing cuda-gdb doesn't really help. However, at this point I say we just merge this to unblock CI since we have incoming fixes in #14292.